data/actions/actionsRouter.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const Actions = require("../data/helpers/actionModel");
+
+const router = express.Router();
+
+router.get("/", (req, res, next) => {
+  Actions.get()
+    .then(actions => {
+      res.status(200).json(actions);
+    })
+    .catch(next);
+});
+
+router.get("/:id", validateActionId, (req, res) => {
+  res.status(200).json(req.action);
+});
+
+router.put(
+  "/:id",
+  validateActionId,
+  validateActionsAtLeastOneBody,
+  (req, res, next) => {
+    const { description, notes, completed } = req.body;
+    Actions.update(req.action.id, { description, notes, completed })
+      .then(action => {
+        res.status(200).json(action);
+      })
+      .catch(next);
+  }
+);
+
+router.delete("/:id", validateActionId, (req, res, next) => {
+  Actions.remove(req.action.id)
+    .then(() => {
+      res.status(200).json(req.action);
+    })
+    .catch(next);
```

Comment:
The catch block should call a function that takes in an error property and either console.logs it (in development) or sends back a status 500 response.